integreat_cms/core/settings.py

```diff
@@ -922,8 +922,11 @@
     },
 }
 
+#: whether redis is activated
+REDIS_CACHE = bool(strtobool(os.environ.get("INTEGREAT_CMS_REDIS_CACHE", "False")))
```

Comment:
Do you think you could move this up, so we could already profit from this in the following line?
https://github.com/digitalfabrik/integreat-cms/blob/78fbad03983656514b245ca58080cab34a326ce3/integreat_cms/core/settings.py#L311-L312

Otherwise we might get inconsistencies when we use e.g. the string "False" as environment variable, since `cacheops` will be enabled even if redis is disabled.

Reply:
Done 🙂 I hope this looks okay.